src/web/mjs/connectors/TenshiMoe.mjs

```diff
@@ -0,0 +1,154 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class TenshiMoe extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'tenshimoe';
+        super.label = 'Tenshi.moe';
+        this.tags = [ 'anime', 'english'];
+        this.url = 'https://tenshi.moe';
+        this.config = {
+            format: {
+                label: 'Language Settings',
+                description: 'Choose between Subbed or Dubbed anime. If your choice is not available the other is used as fallback',
+                input: 'select',
+                options: [
+                    { value: 'SUB', name: 'Subbed' },
+                    { value: 'DUB', name: 'Dubbed' }
+                ],
+                value: ''
+            },
+            resolution: {
+                label: 'Preferred Resolution',
+                description: 'Try to download video in the selected resolution.\nIf the resolution is not supported, depending on the mirror the download may fail, or a fallback resolution may be used!',
+                input: 'select',
+                options: [
+                    { value: '', name: 'Mirror\'s Default' },
+                    { value: '360', name: '360p' },
+                    { value: '480', name: '480p' },
+                    { value: '720', name: '720p' },
+                    { value: '1080', name: '1080p' }
+                ],
+                value: ''
+            }
+        };
+
+    }
+    async _initializeConnector() {
+        //bypass DdoSS Guard
```

Comment:
Probably better to update [Request.mjs](https://github.com/manga-download/hakuneko/blob/1e5953e4d189ded736a50d8bf9b96520aa4c3d6b/src/web/mjs/engine/Request.mjs#L138-L142) so it works for all websites

Reply:
Exactly, i was wondering where to make the change tbh.

Reply:
Done.